src/components/ebay-toast/README.md

```diff
@@ -0,0 +1,22 @@
+# ebay-toast
+
+```marko
+<ebay-toast open a11y-close-text="Close Toast">
+  <h1>Hello World</h1>
+</ebay-toast>
+```
+
+## Attributes
+
+Name | Type | Stateful | Required | Description
+--- | --- | --- | --- | ---
+`open` | Boolean | Yes | No | Whether toast is open.
+`focus` | String | No | No | An id for an element which will receive focus when the toast opens (defaults to close button).
+`a11y-close-text` | String | No | Yes | A11y text for close button and mask.
+
+## Events
+
+Event | Data | Description
+--- | --- | ---
+`toast-show` |  | toast opened
```

Comment:
Not sure if we are naming these events consistently elsewhere, but to me it should be show/hide or open/close. One the other and not mixed.

Reply:
On dialog the events are show and close, which is why I labeled them like that. 
We should change that in next refactor

Reply:
Ok, I'll make a note of it, thanks.